composer.json

```diff
@@ -9,7 +9,14 @@
         "php": "^7.3.0",
         "ext-ctype": "*",
         "ext-iconv": "*",
+        "ext-intl": "*",
         "ext-json": "*",
+        "ext-ldap": "*",
+        "ext-mbstring": "*",
```

Comment:
We don't use MBstring directly, but lost of our vendors do. Not sure if we should keep it here, because Pelagos doesn't require it, but it does need to be installed for the vendor stuff. They however do not specify, and I'm not sure if Composer would break if they had.